tests/admin/test_forms.py

```diff
@@ -240,3 +255,25 @@ def test_raise_error_missing_get_edx_api_data(self, *args):  # pylint: disable=u
         with raises(NotConnectedToOpenEdX) as excinfo:
             get_all_catalogs(None)
         assert message == str(excinfo.value)
+
+    @mock.patch('enterprise.course_catalog_api.get_catalog_api_client')
+    @mock.patch('enterprise.course_catalog_api.CatalogIntegration')
+    @mock.patch('enterprise.course_catalog_api.get_edx_api_data')
+    def test_clean_valid_email(self, mocked_get_edx_data, *args):  # pylint: disable=unused-argument
+        """
+        Tests that valid emails are kept while cleaning email field
```

Comment:
My bad; I used the previous form cleaning method as a template and forgot to change some things.